src/converters/audio/native_audio_converter.ml

```diff
@@ -50,3 +50,14 @@ let samplerate_converter () =
 
 let () =
   Audio_converter.Samplerate.converters#register "native" samplerate_converter
+
+let channel_layout_converter src dst data =
+  assert (src <> dst);
+  match dst with
+    | `Mono -> [| Audio.to_mono data |]
+    | `Stereo -> [| data.(0); data.(0) |]
```

Comment:
What if the input is 5.1?

Reply:
We'll delegate to ffmepg for that. I'll implement it next.